Update/_tsum_001.txt

```diff
@@ -1331,23 +1331,23 @@ void main()
 
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　あの２人は水鉄砲の銃撃戦に相当の熟練があった。",
-		   NULL, "After all, the two of them had considerable experience in water gun battles.", GetGlobalFlag(GLinemodeSp));
```

Comment:
@Norgus And about those 2 sentences as well.

Reply:
I feel like the tense was right to begin with, but honestly I'm not a writer and I don't have the patience to do editing.

Reply:
Yeah, I don't know either. I had reverted what I wrote, but I still have doubts